katello/katello/backup.rb

```diff
@@ -159,6 +159,7 @@ def snapshot_backup
       @snapsize = @options[:snapshot_size] || "2G"
       FileUtils.mkdir_p @mountdir
       confirm unless @options[:confirm]
+      validate_logical_volume unless @options[:confirm]
```

Comment:
There is a commit that is needed prior to this that actually adds this function to the file. This one -- http://projects.theforeman.org/issues/22418